posthog/src/test/java/com/posthog/PostHogTest.kt

```diff
@@ -1426,6 +1427,40 @@ internal class PostHogTest {
         sut.close()
     }
 
+    @Test
```

Comment:
we had tests for $session_id, but not for snapshot events specifically which was a special case